Debug App/Sources/View Controllers/MerchantHeadlessCheckoutAvailablePaymentMethodsViewController.swift

```diff
@@ -330,9 +330,13 @@ extension MerchantHeadlessCheckoutAvailablePaymentMethodsViewController {
         self.logs.append(#function)
         self.primerError = err
         self.hideLoadingOverlay()
-        
-        let rvc = MerchantResultViewController.instantiate(checkoutData: self.checkoutData, error: self.primerError, logs: self.logs)
-        self.navigationController?.pushViewController(rvc, animated: true)
+        if navigationController?.children.last is MerchantHeadlessCheckoutBankViewController {
```

Comment:
Is this something that a merchant will need to do, or is it specific to our implementation?

Reply:
No, specific to our UI implementation based on headless , which requires an additional dismiss of the list of banks at failure - it was more about being aligned with Android/drop-in.